examples/test-c-client.c

```diff
@@ -34,6 +34,11 @@ int main(int argc, char** argv)
     int yes = 1;
     const char message [] = "This message should be sent to the other side";
 
+    if (argc < 3) {
```

Comment:
Better `argc != 3`?

Reply:
its no big deal seen we are not trying to reach argv[>=3]

Reply:
I mean, providing more that 2 arguments means that the usage is most probably incorrect, therefore better to show help. Otherwise further errors may follow, like missing port or else.